relayer/src/link/pending.rs

```diff
@@ -124,12 +124,24 @@ impl<Chain: ChainHandle> PendingTxs<Chain> {
         Ok(Some(all_events))
     }
 
-    // Try and process one pending transaction if available.
+    fn relay_path(&self) -> String {
+        format!(
+            "{}:{}/{} -> {}",
+            self.counterparty_chain_id,
+            self.port_id,
+            self.channel_id,
+            self.chain_id()
+        )
```

Comment:
All these could rather be in KV pairs to potentially aid log filtering.

Reply:
I guess they could but they're also useful when combing through the logs "by hand". But perhaps we can have both?

Reply:
> But perhaps we can have both?

I'm not sure what's the best way to go about achieving this.